content/docs/configuration.md

```diff
@@ -37,7 +37,7 @@ Both Packit-as-a-Service and packit tool use this configuration file.
  `jobs`                    | list of dicts   | a list of job definitions for packit service: see [below](#supported-jobs) for details
  `allowed_gpg_keys`        | list of strings | a list of gpg-key fingerprints; if specified, one of the configured keys have to sign the last commit when updating in downstream; add GitHub key (`4AEE18F83AFDEB23`) if you want to use this on code merged via GitHub web interface
  `spec_source_id`          | int or string   | Numeric ID of Source inside spec file which packit should change when setting path to the newly generated tarball, can be also full name of the macro
-
+ `upstream_tag_template`   | string          | template to create an upstream tag name (upstream may use different tagging scheme), e.g.: "v{version}"
```

Comment:
please tell explicitly that packit will fill in the `version` argument and that it's using the python string formatting